src/components/constraints/ConstraintForm.svelte

```diff
@@ -167,10 +210,14 @@
           class="st-select w-100"
           disabled={constraintPlanId !== null}
           name="model"
+          use:permissionHandler={{
```

Comment:
Any idea why there isn't a tooltip on this model select when the user doesn't have permission to edit the constraint? The other inputs have tooltips but this one doesn't.

Reply:
Is there a plan also selected? If there is a plan selected, the model select becomes actually disabled and thus would not get a hover state.

Reply:
Ah yep plan selected as well, makes sense.